base_partner_merge/base_partner_merge.py

```diff
@@ -1,11 +1,7 @@
 # -*- coding: utf-8 -*-
 # © 2016 Camptocamp SA
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
-from openerp.addons.crm.base_partner_merge import *  # noqa
-
-
-class NoCRMResPartner(ResPartner):  # noqa
-    _module = 'base_partner_merge'
+from openerp.addons.crm.wizard.base_partner_merge import *  # noqa
```

Comment:
@alan196 Put models in a 'models' directory
See https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#directories